docs/src/content/docs/challenges/angular/3-directive-enhancement.md

```diff
@@ -2,7 +2,8 @@
 title: 🟠 Directive Enhancement
 description: Challenge 3 is about enhancing a built-in directive
 author: thomas-laforge
-challengeNumber: 3
+challenge:
+  number: 3
```

Comment:
why did you change `challengeNumber` into `challenge.number` ?

Reply:
It was before I found how to use starlight labels object, I was sending the `challengeLabel` for the Spanish pages. When I implemented the `labels`, I left it as an object in case more information could be sent related to the challenge (maybe a custom icon, image, or something else)

Reply:
Can you undo this change please ? We will change it if need to do it.

Reply:
Updated.